src/core/components/ChatInput.tsx

```diff
@@ -222,12 +230,31 @@ const ChatInput = ({ setShouldAutoScroll }: ChatInputProps) => {
 
       const reader = new FileReader();
       reader.onload = async (e) => {
+        console.log(e.target?.result);
         if (e.target && typeof e.target.result === 'string') {
           const imgID = await saveImage(e.target.result);
           setImageIDs((prevIDs) => [...prevIDs, imgID]);
+        } else if (e.target?.result instanceof ArrayBuffer) {
+          const buf = new Uint8Array(e.target.result);
+          const doc = await library.loadDocument(buf);
+
+          const page = doc.getPage(0);
+          const image = await page.render({ scale: 3, render: 'bitmap' });
```

Comment:
bitmap is the only possible option for the browser, we must take that and use canvas to convert to a base64image url.  sharp library may be used to directly convert to a png or possibly other formats but this is not possible to use in browsers.